### PR TITLE
GitHub Authentication

### PR DESCRIPTION
**Pull request by Lilainita**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "dotenv": "^16.0.1"
```
Comment (manuelq12) [inline package.json:32]: Why is this dependency?

File: test/GithubApi.Authentication.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+describe('Github Api Test', () => {
+    describe('Authentication', () => {
+        it('Via OAuth2 Tokens by Header', async () => {
+            const response = await axios.get(`${urlBase}/repos/${githubUserName}/${repository}`, {
+                headers: {
+                    Authorization: `token ${process.env.ACCESS_TOKEN}`
+                }
+            });
+            //console.log(response)
```
Comment (manuelq12) [inline test/GithubApi.Authentication.test.js:19]: You can erase this comment

**Review by manuelq12 — CHANGES_REQUESTED** [review]
There are some conflict in the package-lock.json that need to be resolved

File: package.json (hunk 1 above)
Comment (Lilainita) [reply package.json:32]: Which dependency? dotenv?

Comment (manuelq12) [reply package.json:32]: Yes

Comment (Lilainita) [reply package.json:32]: I had to install it in order to use the .env file to store the Access Token

Comment (manuelq12) [reply package.json:32]: I see

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**